stable/sentry/values.yaml

```diff
@@ -110,15 +114,43 @@ ingress:
   #     hosts:
   #       - sentry.local
 
+  ## or, if using tlsSecret, enumerate the cert hostnames here
+  tlsHosts: []
+
+## tlsSecret allows a cert to be used for ingress
+#  consider putting this w/ other secrets in seperate encrypted values file
+# tlsSecret:
+#    key:  |
+#    ---BEGIN
+#    ...
+#    ---END
+#    crt: |
+#    ...
+
 # TODO: add support for plugins https://docs.sentry.io/server/plugins/
 
+## allow these chart requirements to be omitted
+## and replaced with external services (using remoteHost)
+requiredBackend:
+    postgresql: true
+    redis: true
```

Comment:
Isn't it easier to just check if `redis.remote` is set? By the way the README hasn't been updated in accordance.